bin/s2nc.c

```diff
@@ -273,11 +277,12 @@ int main(int argc, char *const *argv)
         {"corked-io", no_argument, 0, 'C'},
         {"tls13", no_argument, 0, '3'},
         {"keyshares", required_argument, 0, 'K'},
+        {"non-blocking", no_argument, 0, 'B'},
```

Comment:
Nit: the letter `B` represents blocking more than Non-blocking. Is the letter `N` already taken?

Reply:
Makes sense. I removed 'B' as a short option, so the user must specify --non-blocking.